content/fr/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -0,0 +1,136 @@
+---
+reviewers:
+  - bgrant0607
+  - mikedanese
+  - jygastaud
```

Comment:
Why include these reviewers? They are not in the sig-docs-fr-* OWNERS file.

Reply:
Good catch, I forgot to change it. They came from the EN file.
Will open a new PR with the change.

Reply:
Perfect, thanks for that 🙂